LandSandBoat/server
# Multihit WS & OaT jump audit

**_I affirm:_**

## What does this pull request do?

Audits usage of multihit WS such as follows:

Mainhand, offhand, mainhand extra WS hits (>1 hit WS), mainhand DA/TA/QA/etc procs, offhand DA/TA/QA/etc procs
All WS can proc DA/TA/QA etc twice, but the second proc requires >= 2 base hits on WS or an offhand
Offhand hits now use offhand hit damage on DA/TA/QA etc
WSD no longer applies to Jump.
Skillups on offhand hand now use the appropriate skill if different.

## Steps to test these changes

Use weaponskills, see stuff look pretty normal.
Use Jump and see DA/TA/QA and OaX work properly with normal TP returns.
